WeakAuras/Transmission.lua

```diff
@@ -1677,6 +1676,7 @@ function WeakAuras.Import(inData, target)
     local received = StringToTable(inData, true)
     if type(received) == 'string' then
       -- this is probably an error message from LibDeflate. Display it.
+      wipe(pendingData)
```

Comment:
So, the clean up happens in a hook to OnHide, which is not called in that case because the tooltip is already visible.  (Note that your fix is also incomplete, there's more cleanup to be done.) 

We have a few options:

* We should at least move the cleanup into ShowTooltip, e.g. either directly calling the cleanup, or indirectly via HideUIPanel

This solves all problems where we show the ItemRefTooltip. But if i recall correctly ItemRefTooltip is a global shared tooltip?

So:
* We might need to look at what other methods we might need to hook with a cleanup,
* Or figure out a way to not use a global shared tooltip.

Reply:
ShowTooltip happens after ShowDisplayTooltip in the case of a good import. So we can’t move the cleanup there without also changing when data is populated, or otherwise restructuring (otherwise we’d cleanup the very data we want to use).

What code did I forget to include?

Reply:
I am pretty sure I did mention in the commit message that this isn't a great fix. But anyways, that extra code isn't necessary in this case because ShowTooltip actually handles the case, so long as pendingData is empty.

Since this is such a minor issue, I think I will retract the fix for now and look for a better solution, so that the multiselect can be merged.